apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -141,12 +145,19 @@ class EditSectionForm extends Component {
       handleClose,
       assignedUnitLessonExtrasAvailable,
       assignedUnitTextToSpeechEnabled,
+      assignedUnitRequiresVerifiedInstructor,
       assignedUnitName,
       localeCode,
       isNewSection,
-      showLockSectionField // DCDO Flag - show/hide Lock Section field
+      showLockSectionField,
+      isVerifiedInstructor // DCDO Flag - show/hide Lock Section field
     } = this.props;
 
+    const courseDisplayName =
+      section && section.courseOfferingId
+        ? courseOfferings[section.courseOfferingId].display_name
+        : '';
```

Comment:
Just curious, is it possible to view the edit section portion of the dashboard if there's no section?

Reply:
seems like no, we assume it exists elsewhere, I'll update the check to be less complicated